frontend/.env.development

```diff
@@ -9,8 +9,9 @@ NEXT_PUBLIC_MATOMO_URL=https://matomo.ubreakit.com
 NEXT_PUBLIC_GA_KEY=UA-30506-1
 # If we want to test Google Analytics in dev DEBUG should be true and URL should be
 # https://www.google-analytics.com/analytics_debug.js
-NEXT_PUBLIC_GA_DEBUG=false
+NEXT_PUBLIC_GA_DEBUG=true
```

Comment:
You need to set this in `.env.production` too so that the previews don't dump data into product GA. Then in the one production deployment, we will override the env var

Reply:
>then in the one production deployment, we will override the env var

Could you expound a bit? Is there a third place for env vars? Is there a UI in vercel?

Reply:
Yea, sorry for my brevity. There is a UI on vercel here to override env vars: https://vercel.com/ifixit/react-commerce-prod/settings/environment-variables

![image](https://github.com/iFixit/react-commerce/assets/52104630/ebf84dcb-0987-4b44-9ed2-e4d1da51534b)

Vercel uses `.env.production` for the previews and the production deployment. If we need more granular control than that, we've been putting the preview version of the env var in `.env.production`, and overriding the production value in Vercel's UI.

For example, we only want the production deployment to report to production Sentry. So we put the dev sentry url in `.env.production`, and override it in vercel's ui for the production deployment.

![image](https://github.com/iFixit/react-commerce/assets/52104630/249131e5-57fc-4020-8247-7a9eb215aa56)

Reply:
We also use the vercel ui for env vars we'd like to keep private and not commit to a public repo